bblfsh-dashboard/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for bblfsh-dashboard
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: bblfsh/dashboard
+  tag: v0.1.1
+  pullPolicy: IfNotPresent
+service:
+  name: bblfsh-dashboard
+  ## port where the dashboard will be listen at the LoadBalancer.
+  port: 80
+  ## loadBalancerIP if not set an ephermeral IP will be used.
+  loadBalancerIP:
+settings:
+  ## serverAddr is the address to a bblfsh server, if empty a bblfsh server
+  ## will be deployed by this chart.
```

Comment:
I would mention that it would be deployed in the same pod as the dashboard, so if multiple replicas are configured multiple bblfsh servers will be deployed